pysma/const.py

```diff
@@ -356,22 +697,94 @@
         SENSOR_OPTIMIZER_TEMP,
     ],
     ENERGY_METER_VIA_INVERTER: [
+        SENSOR_GRID_POWER_SUPPLIED,
+        SENSOR_GRID_POWER_ABSORBED,
+        SENSOR_GRID_TOTAL_YIELD,
+        SENSOR_GRID_TOTAL_ABSORBED,
```

Comment:
These sensors are already in `DEVCLASS_INVERTER`. As soon as an Energy Meter is detected it will merge these two lists of sensors.
They should either be removed in `DEVCLASS_INVERTER` or `ENERGY_METER_VIA_INVERTER`.
I do not have an Energy Meter and do not receive any values on these sensors. Do you know if these are Energy Meter only sensors?